diff --git a/out/zero-cache/src/db/lite-tables.js b/out/zero-cache/src/db/lite-tables.js
index fd737b82f10476d4f755b66c4b4009389827ad56..6fc01167ce503e68759880376b7707df31c34bac 100644
--- a/out/zero-cache/src/db/lite-tables.js
+++ b/out/zero-cache/src/db/lite-tables.js
@@ -101,6 +101,7 @@ export function computeZqlSpecs(lc, replica, tableSpecs = new Map()) {
         }
         uniqueColumns.get(tableName)?.push(Object.keys(columns));
     }
+    console.log('UNIQUE COLUMNS', uniqueColumns);
     listTables(replica).forEach(fullTable => {
         // Only include columns for which the mapped ZQL Value is defined.
         const visibleColumns = Object.entries(fullTable.columns).filter(([_, { dataType }]) => dataTypeToZqlValueType(dataType));
